src/Xml/XmlRuntime.fs

```diff
@@ -86,7 +86,7 @@ type XmlElement =
   static member Create(reader:TextReader) =    
     use reader = reader
     let text = reader.ReadToEnd()
-    let element = XDocument.Parse(text).Root 
+    let element = XDocument.Parse(text, LoadOptions.PreserveWhitespace).Root 
```

Comment:
You also need to change CreateList

Reply:
Hey @ovatsus Sorry for the delay. Thanks I've changed that and added tests now.